### PR TITLE
Fixed velocity and can create the name tokens package and class.

### PR DESCRIPTION
**Pull request by branflake2267**
Progress report... Can now create the nametokens package and class body. 

I can move onto the rest. I think I only have one more item to figure out and that will be adding elements, or tweaking existing classes, such as the modules and nametokens. The rest I'm hoping will be repetitive actions I've done already.

### CODE REVIEW

File: .idea/libraries/commons_configuration_commons_configuration_1_9.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    </JAVADOC>
+    <SOURCES>
+      <root url="jar://$PROJECT_DIR$/lib/commons-configuration-1.9-sources.jar!/" />
+      <root url="jar://$PROJECT_DIR$/lib/commons-lang-2.6-sources.jar!/" />
+      <root url="jar://$PROJECT_DIR$/lib/commons-logging-1.1.1-sources.jar!/" />
+      <root url="jar://$PROJECT_DIR$/lib/servlet-api-2.4-sources.jar!/" />
+      <root url="jar://$PROJECT_DIR$/lib/xml-apis-1.0.b2-sources.jar!/" />
+    </SOURCES>
+  </library>
+</component>
```
Comment (branflake2267) [inline .idea/libraries/commons_configuration_commons_configuration_1_9.xml:24]: auto generated file

File: src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterAction.java [hunk 2]
```diff
@@ -167,15 +174,14 @@ private PsiClass createNewNameTokensFile() throws Exception {
...
         RenderedTemplate rendered = createdNameToken.getNameTokensFile();
-        String name = rendered.getNameAndNoExt();
-        // TODO add contents
+        String nameFile = rendered.getNameAndNoExt();
         String contents = rendered.getContents();
 
-        PsiDirectory dir = createdNameTokensPackage.getContainingFile().getContainingDirectory();
+        PsiDirectory[] createdNameTokensPackageDirectories = createdNameTokensPackage.getDirectories();
+        PsiFile element = PsiFileFactory.getInstance(project).createFileFromText(nameFile, JavaFileType.INSTANCE, contents);
+        PsiElement createdNameTokensClass = createdNameTokensPackageDirectories[0].add(element);
```
Comment (meriouma) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterAction.java]: You'll never get an IndexOutOfBoundsException?

**Comment by meriouma** [comment]
LGTM

File: src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterAction.java (hunk 2 above)
Comment (branflake2267) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterAction.java]: Since I'm creating the class from scratch I would think no.

**Pull request merged**